tests/e2e/gc_metadata_syncer.go

```diff
@@ -385,7 +385,7 @@ var _ = ginkgo.Describe("[csi-guest] pvCSI metadata syncer tests", func() {
 	// 10. Delete PVCs.
 	// 11. Delete SC.
 
-	ginkgo.It("Statefulset tests with label updates", func() {
+	ginkgo.It("Statefulset tests with label updates", ginkgo.Label(p0, block, tkg, vc70), func() {
```

Comment:
Can be considered as P1,
On standalone PVC  POD same case is already tested. This is similar case on statefullset.